data/mods/Magiclysm/monsters/golems.json

```
@@ -122,5 +119,35 @@
     "special_attacks": [ { "type": "spell", "spell_data": { "id": "gas_attack", "min_level": 5 }, "cooldown": 60 } ],
     "death_function": { "corpse_type": "BROKEN" },
     "flags": [ "SEES", "NO_BREATHE", "ACIDPROOF", "LOUDMOVES" ]
+  },
+  {
+    "id": "mon_eshaper_golem",
+    "type": "MONSTER",
+    "name": "earthshaper's golem",
+    "description": "This golem is easy to separate from other golems - the shiny earthshaper's emblem shine on his smooth body, and his eyes and hands are made out of purple crystals.",
```

```suggestion
    "description": "This golem is easy to separate from other golems - the metallic earthshaper's emblem shines on its smooth body, and it's eyes and hands are made out of amethyst.",
```
I don't think golems have a gender.